packages/react/src/hooks/useAnchoredPosition.ts

```diff
@@ -30,22 +31,49 @@ export function useAnchoredPosition(
   const floatingElementRef = useProvidedRefOrCreate(settings?.floatingElementRef)
   const anchorElementRef = useProvidedRefOrCreate(settings?.anchorElementRef)
   const [position, setPosition] = React.useState<AnchorPosition | undefined>(undefined)
+  const [_, setPrevHeight] = React.useState<number | undefined>(undefined)
```

Comment:
used to force the height of the selectPanel when filtered and it wants to shrink but it's anchored at the top (we want it to stay at the top and lock the height)